### PR TITLE
[MAINT, MRG] Move `_check_mri` to _freesurfer.py

### PR DESCRIPTION
**Pull request by alexrockhill**
Puts `_check_mri` in a more appropriate module.

### CODE REVIEW

File: mne/viz/misc.py [hunk 1]
```diff
@@ -520,6 +519,7 @@ def plot_bem(subject=None, subjects_dir=None, orientation='coronal',
     on top of the midpoint MRI slice with the BEM boundary drawn for that
     slice.
     """
+    from .._freesurfer import _check_mri
```
Comment (larsoner) [inline mne/viz/misc.py:522]: It doesn't seem like this should need to be nested, does it actually need to be?

**Review by larsoner — COMMENTED** [review]
Otherwise LGTM

File: mne/viz/misc.py (hunk 1 above)
Comment (alexrockhill) [reply mne/viz/misc.py:522]: The other one is, I'll pull them both out

File: mne/viz/misc.py [hunk 2]
```diff
@@ -309,7 +309,6 @@ def _plot_mri_contours(mri_fname, surfaces, src, orientation='coronal',
     """Plot BEM contours on anatomical slices."""
     import matplotlib.pyplot as plt
     from matplotlib import patheffects
-    from .._freesurfer import _mri_orientation, _read_mri_info
```
Comment (larsoner) [inline mne/viz/misc.py:312]: This is better, internal MNE imports should be nested only when required by import cycle problems

**Review by larsoner — APPROVED** [review]

**Review by agramfort — APPROVED** [review]

**Comment by larsoner** [comment]
Thanks @alexrockhill !

**Pull request merged**